test/coverlet.core.tests/Instrumentation/InstrumenterTests.cs

```diff
@@ -20,8 +20,7 @@ namespace Coverlet.Core.Instrumentation.Tests
 {
     public class InstrumenterTests
     {
-        private readonly InstrumentationHelper _instrumentationHelper = new InstrumentationHelper(new ProcessExitHandler(), new RetryHelper(), new FileSystem());
-        private readonly Mock<ILogger> _mockLogger = new Mock<ILogger>();
```

Comment:
why remove this mock?

Reply:
Because I don't see the point in using the same mock instance in the different tests. Actually I think it is more realistic to pass a new mocked logger instance per test. I also think the tests are more easy to read this way. 
But I also don't mind changing it back.

Reply:
I prefer to duplicate code only when will behave differently, in this case every test will behave in the same way(do nothing), do you mind revert the mock part?

Reply:
Done.